kivy/core/window/__init__.py

```diff
@@ -528,6 +528,18 @@ def _get_system_size(self):
     '''Real size of the window ignoring rotation.
     '''
 
+    def get_effective_size(self):
```

Comment:
There is no need for this to be exposed to the user, this is used internally only and thus should start with a `_` `_get_effective_size`.